source/operations/external-iam/configure-ad-ldap-external-identity-management.rst

```diff
@@ -150,99 +196,62 @@ An AD/LDAP user with no assigned policy *and* with membership in groups with no
+   You must restart the MinIO deployment to apply the configuration changes.
+
+   If you configured AD/LDAP from the MinIO Console, no additional action is required.
+   The MinIO Console automatically restarts the deployment when the new AD/LDAP configuration is saved.
+
+   For MinIO Client and environment variable configuration, use the :mc-cmd:`mc admin service restart` command to restart the deployment:
 
    .. code-block:: shell
       :class: copyable
 
       mc admin service restart ALIAS
 
-   Replace ``ALIAS`` with the :ref:`alias <alias>` of the deployment to 
-   restart.
+   Replace ``ALIAS`` with the :ref:`alias <alias>` of the deployment to restart.
 
    3) Use the MinIO Console to Log In with AD/LDAP Credentials
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   The MinIO Console supports the full workflow of authenticating to the
-   AD/LDAP provider, generating temporary credentials using
-   the MinIO :ref:`minio-sts-assumerolewithldapidentity` Security Token Service
-   (STS) endpoint, and logging the user into the MinIO deployment.
-
-   Starting in :minio-release:`RELEASE.2021-07-08T01-15-01Z`, the MinIO Console is
-   embedded in the MinIO server. You can access the Console by opening the root URL
-   for the MinIO cluster. For example, ``https://minio.example.net:9000``.
+   The MinIO Console supports the full workflow of authenticating to the AD/LDAP provider, generating temporary credentials using the MinIO :ref:`minio-sts-assumerolewithldapidentity` Security Token Service (STS) endpoint, and logging the user into the MinIO deployment.
 
-   From the Console, click :guilabel:`BUTTON` to begin the Active Directory / LDAP
-   authentication flow.
+   Starting in :minio-release:`RELEASE.2021-07-08T01-15-01Z`, the MinIO Console is embedded in the MinIO server.
+   You can access the Console by opening the root URL for the MinIO cluster. For example, ``https://minio.example.net:9000``.
 
-   Once logged in, you can perform any action for which the authenticated
-   user is :ref:`authorized 
-   <minio-external-identity-management-ad-ldap-access-control>`. 
+   Once logged in, you can perform any action for which the authenticated user is :ref:`authorized <minio-external-identity-management-ad-ldap-access-control>`.
 
-   You can also create :ref:`access keys <minio-idp-service-account>` for
-   supporting applications which must perform operations on MinIO. Access Keys
-   are long-lived credentials which inherit their privileges from the parent user.
-   The parent user can further restrict those privileges while creating the service
-   account. 
+   You can also create :ref:`access keys <minio-idp-service-account>` for supporting applications which must perform operations on MinIO.
+   Access Keys are long-lived credentials which inherit their privileges from the parent user.
+   The parent user can further restrict those privileges while creating the service account.
```

Comment:
I think we need to be clear that `service account` is the same as `access keys`.
Otherwise, this is a jarring switch when the rest of the paragraph is talking about access keys.

But I think we call them `service accounts` in most places in MinIO, so maybe it's worth intentionally linking the two.

Reply:
The page this links to says `service account` is an outdated name for `access key`. Which suggests favoring `access key`, like what `mc admin user svcacct` does. (Except, of course, all the _other_ places that use `service account`.)

This is confusing _for me_, much less a novice docs reader.

Is there an established way to differentiate between "an access key I, a human, uses to authenticate" and "an access key for an automated service?"

Reply:
With all of the other `mc` command reworking that's happened, maybe move this functionality to `mc admin user accesskey`. Or even just pull it out to `mc admin accesskey`. But that's not something done in the docs.

As long as `svcacct` is the command, I don't think we can avoid the confusion.

Reply:
I think I raised this to @kannappanr a while back, but I'm not sure if/when we will have engineering resources to add it as an alias (effectively).